src/usr/local/containerbase/tools/v2/elixir.sh

```diff
@@ -8,12 +8,14 @@ function install_tool () {
 
   check_command erl
 
-
   # https://github.com/elixir-lang/elixir/releases/tag/v1.14.0
-  if [ "$MAJOR" -eq 1 ] && [ "$MINOR" -eq 14 ]; then
-    base_file=elixir-otp-23.zip
-  elif [ "$MAJOR" -eq 1 ] && [ "$MINOR" -gt 14 ]; then
+  # https://hexdocs.pm/elixir/compatibility-and-deprecations.html#between-elixir-and-erlang-otp
+  if [ "$MAJOR" -eq 1 ] && [ "$MINOR" -ge 17 ]; then
+    base_file=elixir-otp-27.zip
```

Comment:
can't we use the lowest version instead, so it's compatible with more erlang major versions? I'll   refactor this code soon to use the current installed erlang major as base and install the most appropriate elixir version.

Reply:
You're right. I was misunderstanding Elixir builds by thinking they were compiled for a specific OTP version and therefore were incompatible with any other OTP version.

- https://elixir-lang.org/install.html#precompiled-package
- https://github.com/hexpm/bob?tab=readme-ov-file#elixir-builds